ui/app/components/job-status/panel/steady.js

```diff
@@ -217,6 +217,13 @@ export default class JobStatusPanelSteadyComponent extends Component {
     // If all allocs are running, the job is Healthy
     const totalAllocs = this.totalAllocs;
 
+    if (this.job.status === 'dead' && this.job.stopped) {
```

Comment:
I like that we've included both conditions here, because the job status won't be `dead` until the scheduler has had at least once chance at it to stop allocs. Is there a reason not to include that condition in `ui/app/models/job.js`?

Reply:
Great question and I should've explained it in a comment. We don't get job.Status back on the /statuses endpoint (there's a joke here I think) is the only reason; I'll add it now.